src/orders-update.js

```diff
@@ -143,6 +146,7 @@ export default class OrdersUpdate {
                 version: existingOrder.version,
                 actions,
               })
+              .then(result => result.body)
```

Comment:
If there are no changes on an order, script returns `order` object. But when there are some changes it returns API response envelope instead `{ body: order }` so with this we will return an order every time.